Formula/ipmitool.rb

```diff
@@ -16,6 +16,8 @@ class Ipmitool < Formula
     sha256 x86_64_linux:   "a0a993a436ef12c14707d60293d37233dd90e0e80909c461f262c19874951d32"
   end
 
+  deprecate! date: "2022-12-16", because: :repo_archived
```

Comment:
Why this date, though? The last commit was made on 2023-01-12; the archive date should be after that.

https://github.com/ipmitool shows "Updated on Feb 9", so maybe 2023-02-09?

Reply:
```suggestion
  deprecate! date: "2023-02-09", because: :repo_archived
```